lib/components/table.vue

```diff
@@ -76,14 +76,15 @@
 </template>
 
 <script>
-    import {warn} from '../utils';
+    import { warn } from '../utils';
+    import { keys } from '../utils/object';
```

Comment:
Although I think you can only do one nested destructure (but as many sub properties as you want)